sycl/include/sycl/ext/intel/math/imf_bf16.hpp

```diff
@@ -0,0 +1,205 @@
+//==-------------------- imf_bf16.hpp - bfloat16 utils ---------------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// C++ APIs for bfloat16 util functions.
+//===----------------------------------------------------------------------===//
+
+#pragma once
+#include <sycl/ext/oneapi/experimental/bfloat16.hpp>
+#include <type_traits>
+
+using sycl_bfloat16 = sycl::ext::oneapi::experimental::bfloat16;
+using _iml_bfloat16_internal = uint16_t;
```

Comment:
I don't think that can be done in headers. What am I missing?

Reply:
Hi, @aelovikov-intel 
Could you explain a little more about what do you think that can't be done in headers?

Thanks very much.

Reply:
Because it spoils the namespace in a way visible to the customer. I.e., if SYCL doesn't define `::sycl_bfloat16` then we must not introduce it. Also, what if customer had their own "using sycl_bfloat16 = int" at the global level (which is allowed per the spec).

Reply:
Hi, @aelovikov-intel 
I have removed sycl_bfloat16 usage in the patch to avoid polluting sycl namespace.
Thanks very much.